src/synthesizer/java/mod.rs

```diff
@@ -0,0 +1,649 @@
+#![allow(unused_variables)]
+
+use crate::code::{CodeBuffer, IndentOptions};
+use crate::ir::CloudformationProgramIr;
+use crate::ir::importer::ImportInstruction;
+use crate::ir::resources::ResourceIr;
+use crate::ir::conditions::ConditionIr;
+use crate::parser::lookup_table::MappingInnerValue;
+use crate::specification::{CfnType, Structure};
+use std::any::{type_name};
+use std::borrow::Cow;
+use std::fmt::{Debug};
+use std::io;
+use std::rc::Rc;
+use super::Synthesizer;
+use voca_rs::case::camel_case;
+use crate::ir::outputs::OutputInstruction;
+use crate::ir::reference::{Origin, PseudoParameter, Reference};
+
+const INDENT: Cow<'static, str> = Cow::Borrowed("\t");
```

Comment:
The Google style guide for Java is fairly popular and demands two spaces is used for intent.

```suggestion
const INDENT: Cow<'static, str> = Cow::Borrowed("  ");
```